app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,32 @@
+class FoodsController < ApplicationController
+  # before_action :authenticate_user!
+  include ApplicationHelper
+
+  def index
+    @foods = current_user.foods
```

Comment:
I guess you could say `@foods = Food.all` and do similar to the other ones where you have 'current_user'

![image](https://user-images.githubusercontent.com/68971295/188717092-002cdc3a-b221-4c22-9cb4-ff6e4a882616.png)